tools/twix/src/main.rs

```diff
@@ -319,14 +218,17 @@ impl App for TwixApp {
                         CompletionEdit::select_all(&self.panel_selection, ui, panel_input.id);
                     }
                     if panel_input.changed() || panel_input.lost_focus() {
-                        if let Ok(panel) = SelectablePanel::try_from_name(
+                        match SelectablePanel::try_from_name(
                             &self.panel_selection,
                             self.nao.clone(),
                             None,
                         ) {
-                            if let Some(active_panel) = self.active_panel() {
-                                *active_panel = panel;
+                            Ok(panel) => {
+                                if let Some(active_panel) = self.active_panel() {
+                                    *active_panel = panel;
+                                }
                             }
+                            Err(err) => error!("{err:?}"),
                         }
```

Comment:
What was the reason for this change?

Reply:
Otherwise the error wouldn't have been logged